src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/hideutilityclassconstructor/Example2.java

```diff
@@ -0,0 +1,52 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="HideUtilityClassConstructor">
+      <property name="ignoreAnnotatedBy"
+        value="SpringBootApplication, java.lang.Deprecated" />
+    </module>
+   </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.design.hideutilityclassconstructor;
+
+// xdoc section -- start
+@java.lang.Deprecated
+class Example2 { // ok, skipped by annotation
```

Comment:
please put "// ok, ..." to line where violation supposed to be, a line above.

Reply:
Done